### PR TITLE
Encryptr v2.0 (now a SpiderOak project)

### PR DESCRIPTION
**Pull request by felipebueno**

### CODE REVIEW

File: Casks/encryptr.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 cask :v1 => 'encryptr' do
-  version '1.2.0'
-  sha256 '6216e5185755b8f24b2b0a950e3c1b511df11730dca968bff5e42b3ba0ad9bf8'
+  version '2.0.0'
+  sha256 '54e2ef8f8c4d22a20793b68fd35c65f8af34ffd783c48140b84c1a9b445589ea'
 
   # github.com is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/encryptr.rb]: Comment must be removed.

**Comment by felipebueno** [comment]
@vitorgalvao comment removed :+1:

**Comment by vitorgalvao** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by adityadalal924** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 8ab780922789eb3323a73457d729b1a2d701a1cf. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**